### PR TITLE
chore(snippetz): playground example

### PR DESCRIPTION
**Pull request by tmastrom**
Currently, we don't have a playground example for the `@scalar/snippetz` package in the monorepo. 

This PR migrates the old snippetz demo, containerizes, and deploys it.

### CODE REVIEW

File: packages/snippetz/playground/.gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# Editor directories and files
+.vscode/*
+!.vscode/extensions.json
+.idea
+.DS_Store
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.sw?
```
Comment (hanspagel) [inline packages/snippetz/playground/.gitignore:24]: should be the same as in root, can we delete this file? if something is missing in the root file, let’s add it there

File: packages/snippetz/playground/.vscode/extensions.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (hanspagel) [inline packages/snippetz/playground/.vscode/extensions.json:1]: let’s just use the one in root and delete this here

**Comment by hanspagel** [comment]
AWESOME :) I know you just copied the files, but let’s use the opportunity to clean it up a bit (e.g. extend the tsconfig we use here in most packages)

**Review by hanspagel — APPROVED** [review]
NICE, time to roll this out

File: pnpm-workspace.yaml [hunk 3]
```diff
@@ -2,3 +2,4 @@ packages:
   - 'packages/*'
   - 'examples/*'
   - 'playwright'
+  - 'packages/snippetz/playground'
```
Comment (hanspagel) [inline pnpm-workspace.yaml:5]:
```suggestion
  - 'packages/*'
  - 'packages/snippetz/playground'
  - 'examples/*'
  - 'playwright'
```

**Comment by tmastrom** [comment]
This lockfile is busted. Closing in favour of https://github.com/scalar/scalar/pull/3369 instead of continuing to slog through rebase dependencies.

**Pull request closed without merging**